rir/src/compiler/pir/builder.cpp

```diff
@@ -125,8 +126,9 @@ Builder::Builder(Continuation* cnt, Value* closureEnv)
         }
         auto mkenv = new MkEnv(closureEnv, names, args.data(), miss);
 
-        auto rirCode = cnt->owner()->rirFunction()->body();
-        mkenv->updateTypeFeedback().feedbackOrigin.srcCode(rirCode);
+        // FIXME: what does this mean, we need both rirFun and we need idx
```

Comment:
it doesn't make sense to me. I wuld remove the setter altogether and comment out this line